src/app/services/traveler.service.ts

```diff
@@ -1,4 +1,4 @@
-import { TravelerModel } from './../manage-bookings/models/booking-types';
+import { TravelerModel, TravelerRequest } from './../manage-bookings/models/booking-types';
 import { HttpClient, HttpErrorResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
```

Comment:
why is this at the top level shouldn't it be inside the manage-bookings module

Reply:
you are right. I will make those changes